lib/trino-parquet/src/test/java/io/trino/parquet/writer/TestParquetSchemaConverter.java

```diff
@@ -123,6 +125,55 @@ public void testMapKeyRepetitionLevel()
         assertThat(valueType.isRepetition(OPTIONAL)).isTrue();
     }
 
+    /**
+     * This test ensures the parquet writer matches the parquet spec for lists.
+     *
+     * From https://github.com/apache/parquet-format/blob/master/LogicalTypes.md#nested-types
```

Comment:
Let's change the link to `https://github.com/apache/parquet-format/blob/master/LogicalTypes.md#lists`